### PR TITLE
Spinners customization

### LINKED ISSUE #124: Add customization controls to Spinners page
[Design](https://cdn.zeplin.io/5a5e24188993724b2abfc6a9/screens/15251BCC-0175-4D11-9E52-117DDF586AAC.png)

Use vuestic-slider and vuestic-color picker

### PR DESCRIPTION
**Pull request by vlad-shusterman**
close #124

### CODE REVIEW

File: src/components/ui/spinners/Spinners.vue [hunk 1]
```diff
@@ -28,48 +57,143 @@
...
+      return grouped
     }
   }
+}
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
+
+.spinners {
```
Comment (jean-moldovan) [inline src/components/ui/spinners/Spinners.vue]: empty styles?

File: src/components/ui/spinners/Spinners.vue [hunk 2]
```diff
@@ -28,48 +57,143 @@
...
+      },
+      speed: 1500,
+      paletteArray: colorArray,
+      color: '#4AE387',
+      sliderSize: {
+        formatter: v => `${v}px`,
+        min: 40,
+        max: 100
+      },
+      sliderDuration: {
```
Comment (jean-moldovan) [inline src/components/ui/spinners/Spinners.vue]: durationRange

File: src/components/ui/spinners/Spinners.vue [hunk 3]
```diff
@@ -28,48 +57,143 @@
...
+    groups () {
+      return this.groupItems(Object.keys(spinners), this.config.group)
     },
-    methods: {
-      groupItems (items, groupSize) {
-        let grouped = []
+  },
+
+  watch: {
+    speed: function (speed) {
```
Comment (jean-moldovan) [inline src/components/ui/spinners/Spinners.vue]: Why not computed property?

Comment (vlad-shusterman) [reply src/components/ui/spinners/Spinners.vue]: already fix

**Pull request merged**